### PR TITLE
[6] Product Page using HTML

### PR DESCRIPTION
**Pull request by Israel-Force**
Screenshot (229)
Screenshot (230)
Screenshot (231)
Screenshot (232)
Screenshot (233)

## Task: `HTML task`
- `i added an index.html file that contains solution to the html task`

### CODE REVIEW

**Comment by ToluAkin** [comment]
The screenshots that needs to be added is the output on the web i.e what your page looks like.

File: submissions/israelOladosu/HTML/index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/israelOladosu/HTML/index.html:1]: Remove all styling in this file

**Comment by Israel-Force** [comment]
Thanks for the review.

I have implemented the required changes;
-changed the image
-removed all styling.

File: submissions/israelOladosu/HTML/index.html (hunk 1 above)
Comment (Israel-Force) [reply submissions/israelOladosu/HTML/index.html:1]: I have done that now.

**Pull request closed without merging**